test/Map/Reproject.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+/*global require,describe,it,expect*/
+var Reproject = require('../../lib/Map/Reproject');
+
+describe('Reproject', function() {
+
+    it('function crsStringToCode translates CRS strings to Proj4 codes', function() {
+        expect(Reproject.crsStringToCode("EPSG:4326")).toEqual("EPSG:4326");
+        expect(Reproject.crsStringToCode("EPSG:1234")).toEqual("EPSG:1234");
+        expect(Reproject.crsStringToCode("urn:ogc:def:crs:EPSG:6.6:4326")).toEqual("EPSG:4326");
+        expect(Reproject.crsStringToCode("urn:ogc:def:crs:EPSG:6.6:1241")).toEqual("EPSG:1241");
+        expect(Reproject.crsStringToCode("urn:ogc:def:crs:EPSG::4326")).toEqual("EPSG:4326");
+        expect(Reproject.crsStringToCode("urn:ogc:def:crs:EPSG::1241")).toEqual("EPSG:1241");
+        expect(Reproject.crsStringToCode("CRS84")).toEqual("EPSG:4326");
+    });
+
+    it('function willNeedReprojecting predicts correctly if something needs reprojecting', function() {
+        expect(Reproject.willNeedReprojecting("EPSG:4326")).toBeFalsy();
```

Comment:
I think a stronger assertion makes sense here.  e.g. make it `.toBe(false)` because you want that function to actually return true and false, not just something truthy or falsy.